modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```

Comment:
i guess i'm not sure why we'd want to add ambiguity here.  the method is for sending a receive-side video constraint, so why not name it as such?